recipes/azure-storage-blob/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "azure-storage-blob" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "4fdcdc20e36d0f97a58bdffe1b26fc2b8b983c59ff8625e961c188c925891c66" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - azure-common >=1.1.5
+    - azure-storage-common >=1.1.0,<1.2.0
+
+test:
+  imports:
+    - azure
+    - azure.storage
+    - azure.storage.blob
+
+about:
+  home: https://github.com/Azure/azure-storage-python
+  license: Apache-2.0
```

Comment:
This needs a license file, and a license family.

Reply:
Added both and raised an issue upstream to include the license file in the release tarballs: https://github.com/Azure/azure-storage-python/issues/477